lib/bokken/events/event_admin.ex

```diff
@@ -46,4 +47,20 @@ defmodule Bokken.Events.EventAdmin do
       enrollments_close: nil
     ]
   end
+
+  def send_email(users, email) do
+    users
+    |> Enum.reduce(
+      %{success: [], fail: []},
+      fn user, accumulator ->
+        case Mailer.deliver(email.(user)) do
+          {:ok, _} ->
+            %{success: [user.email | accumulator[:success]], fail: accumulator[:fail]}
+
+          {:error, _} ->
+            %{success: [accumulator[:success]], fail: [user.email | accumulator[:fail]]}
+        end
+      end
+    )
+  end
```

Comment:
Not related to this PR in particular, but I am seeing this function used and duplicated in many different contexts. I think we should create an utils file and put it there. What do you think @RuiL1904?

Reply:
Agreed.